### DISCUSSION TITLE
[bug] Determine whether the file exists before downloading the required file & Repeated downloads by different users & Unable to restart normally when downloading takes too long

### DISCUSSION THREAD

**Discussion by xxxxj-up** [valinet/ExplorerPatcher#1765] [Ideas]
1 Determine whether the file exists before downloading the required file
2 Repeated downloads by different users
3 Unable to restart normally when downloading takes too long

**Comment by pyrates999** [comment]
If this is for symbols downloading, I think explorer patcher already does this: https://github.com/valinet/ExplorerPatcher/wiki/Symbols
